src/gui/newwizard/setupwizardaccountbuilder.h

```diff
@@ -97,6 +97,8 @@ class SetupWizardAccountBuilder
      */
     bool hasValidCredentials() const;
 
+    QString userName() const;
```

Comment:
Is there a long-term plan to return the display name here? `userName` seems ambiguous since it currently returns the DAV user name, but is supposed to return the display name in the future.